package.json

```diff
@@ -29,17 +28,10 @@
     "url": "https://github.com/seegno/jscs-config-seegno/issues"
   },
   "homepage": "https://github.com/seegno/jscs-config-seegno#readme",
-  "dependencies": {
-    "jscs": "^2.5.0"
-  },
   "devDependencies": {
-    "babel": "^5.8.23",
+    "jscs": "^2.8.0",
     "mocha": "^2.3.3",
-    "should": "^7.1.0"
-  },
-  "peerDependencies": {
-    "jscs": ">= 2.5",
```

Comment:
I agree with you on principle, but the fact that npm@3 still handles peerDependencies in a very disturbing way makes me prefer not keeping this here as I don't think anyone will be using a jscs preset without jscs installed. Not even the owner of the project does that - https://github.com/mrjoelkemp/jscs-preset-mrjoelkemp/blob/master/package.json.